pkg/util/utils.go

```diff
@@ -682,18 +682,21 @@ func DefaultContainerConfig() *config.Config {
 	return containerConfig
 }
 
-func CreateCidFile(cidfile string, id string) error {
-	cidFile, err := OpenExclusiveFile(cidfile)
-	if err != nil {
-		if os.IsExist(err) {
-			return fmt.Errorf("container id file exists. Ensure another container is not using it or delete %s", cidfile)
+func CreateIDFile(path string, id string) error {
+	baseDir := filepath.Dir(path)
+	if baseDir != "" {
+		if _, err := os.Stat(baseDir); err != nil {
+			return err
 		}
-		return fmt.Errorf("opening cidfile %s", cidfile)
 	}
+	cidFile, err := os.OpenFile(path, os.O_RDWR|os.O_CREATE|os.O_TRUNC, 0666)
```

Comment:
This can just use os.Create(). Although I wonder why do we create with 0666? I would expect something like 0644 but I guess the umask takes care of that anyway.